services/matrix/matrix.service.js

```diff
@@ -5,9 +5,15 @@ import {
   pathParam,
   queryParam,
 } from '../index.js'
+import { nonNegativeInteger } from '../validators.js'
+
+const fetchModeEnum = ['guest', 'summary']
 
 const queryParamSchema = Joi.object({
   server_fqdn: Joi.string().hostname(),
+  fetchMode: Joi.string()
```

Comment:
thanks for fixing this!
any chance we can still make this snake_case without breaking the world, though? 🙈

Reply:
All the base attributes for shields badges use camel case.
'server_fqdn' should be camel case as well but that would be a breaking change, so that's how we ended up here ;)

Reply:
Bah. I should have picked this up in review.
Tbh, this is more of a codebase-wide mess than that.
I've raised an issue at https://github.com/badges/shields/issues/10804 about how to approach this as a wider issue.